crates/cairo-lint-core/src/lints/bool_comparison.rs

```diff
@@ -0,0 +1,30 @@
+use cairo_lang_defs::plugin::PluginDiagnostic;
+use cairo_lang_diagnostics::Severity;
+use cairo_lang_syntax::node::db::SyntaxGroup;
+use cairo_lang_syntax::node::kind::SyntaxKind;
+use cairo_lang_syntax::node::SyntaxNode;
+
+pub const BOOL_COMPARISON: &str = "Unnecessary comparison with a boolean value. Use the variable directly.";
+
+pub fn check_bool_comparison(db: &dyn SyntaxGroup, node: SyntaxNode, diagnostics: &mut Vec<PluginDiagnostic>) {
+    print!("check_bool_comparison\n");
```

Comment:
if you want to add logs you can use the `log` crate, it's already imported and used somewhere in the `fix.rs` file i think. Also in general don't do `print!("check_bool_comparison\n");` but do `println!("check_bool_comparison");`